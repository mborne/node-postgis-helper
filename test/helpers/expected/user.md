## user

| Name      | Title          | Type                 | Required | Description |
| --------- | -------------- | -------------------- | -------- | ----------- |
| id        | Identifier     | serial               | N        |             |
| username  | Username       | text                 | N        |             |
| birthdate | Birthdate      | date                 | N        |             |
| is_admin  | Administrator? | boolean              | N        |             |
| location  | Location       | geometry(Point,4326) | N        |             |
