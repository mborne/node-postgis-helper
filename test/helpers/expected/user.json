{
  "name": "user",
  "title": "user",
  "description": null,
  "columns": [
    {
      "name": "id",
      "title": "Identifier",
      "description": null,
      "type": "serial",
      "required": false
    },
    {
      "name": "username",
      "title": "Username",
      "description": null,
      "type": "text",
      "required": false
    },
    {
      "name": "birthdate",
      "title": "Birthdate",
      "description": null,
      "type": "date",
      "required": false
    },
    {
      "name": "is_admin",
      "title": "Administrator?",
      "description": null,
      "type": "boolean",
      "required": false,
      "default": false
    },
    {
      "name": "location",
      "title": "Location",
      "description": null,
      "type": "geometry(Point,4326)",
      "required": false
    }
  ],
  "primaryKey": [
    "id"
  ],
  "foreignKeys": []
}